src/pudl/cli/reset_db.py

```diff
@@ -32,17 +35,18 @@ def reset_db(
     if metadata is None:
         metadata = Package.from_resource_ids().to_sql()
 
-    # Check if a SQLite file already exists at the given path, and delete it if it does
     if sqlite_path.exists():
+        logger.info(f"Removing existing DB at {sqlite_path}...")
         sqlite_path.unlink()
 
-    # Use SQLAlchemy to create a new SQLite database at the given path
+    logger.info(f"Creating DB at {sqlite_path}...")
     engine = sa.create_engine(f"sqlite:///{sqlite_path}")
-    conn = engine.connect()
 
     # Use the metadata to initialize the table schema in the new database
-    metadata.create_all(conn)
-    conn.close()
+    with engine.connect() as conn:
+        logger.info(f"Writing schema to DB at {sqlite_path}...")
+        metadata.create_all(conn)
+        conn.close()
```

Comment:
`conn.close()` is called when using the context manager, so you can delete this line.

Reply:
Oh, duh. Thanks for catching this!